From c63c4d00420dcd0da66b4562275699c45441faaa Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Wed, 30 Sep 2020 01:18:08 -0400
Subject: [PATCH 014/148] Revert "FODCircleView: Post calls of main hal
 functions on background thread"

This reverts commit cabd0371c73e3f63e7d7e1ec0f7933614cbe7e13.
---
 .../systemui/biometrics/FODCircleView.java      | 17 ++++-------------
 1 file changed, 4 insertions(+), 13 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java b/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java
index 221c3fd4ffa..33492dffbd7 100644
--- a/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java
+++ b/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java
@@ -60,7 +60,6 @@ import com.android.internal.widget.LockPatternUtils;
 import com.android.keyguard.KeyguardSecurityModel.SecurityMode;
 import com.android.keyguard.KeyguardUpdateMonitor;
 import com.android.keyguard.KeyguardUpdateMonitorCallback;
-import com.android.settingslib.utils.ThreadUtils;
 import com.android.systemui.R;
 import com.android.systemui.Dependency;
 import com.android.systemui.statusbar.policy.ConfigurationController;
@@ -509,9 +508,7 @@ public class FODCircleView extends ImageView implements ConfigurationListener {
         }
 
         setDim(true);
-        ThreadUtils.postOnBackgroundThread(() -> {
-            dispatchPress();
-        });
+        dispatchPress();
 
         updatePosition();
         updateIconDim();
@@ -525,9 +522,7 @@ public class FODCircleView extends ImageView implements ConfigurationListener {
         updateIconDim();
         invalidate();
 
-        ThreadUtils.postOnBackgroundThread(() -> {
-            dispatchRelease();
-        });
+        dispatchRelease();
         setDim(false);
 
         setKeepScreenOn(false);
@@ -599,9 +594,7 @@ public class FODCircleView extends ImageView implements ConfigurationListener {
 
         updatePosition();
 
-        ThreadUtils.postOnBackgroundThread(() -> {
-            dispatchShow();
-        });
+        dispatchShow();
         setVisibility(View.VISIBLE);
     }
 
@@ -609,9 +602,7 @@ public class FODCircleView extends ImageView implements ConfigurationListener {
         mIsShowing = false;
         setVisibility(View.GONE);
         hideCircle();
-        ThreadUtils.postOnBackgroundThread(() -> {
-            dispatchHide();
-        });
+        dispatchHide();
     }
 
     private void updateAlpha() {
-- 
2.17.1

